### DISCUSSION TITLE
Is this addon still maintained?

### DISCUSSION THREAD

**Discussion by XinoGitHub** [hassio-addons/addon-unifi#430] [General]
I ask this because from my HA Addon it says "maintained no! (as of 2022), but then on github it says "maintained yes"
Also, latest version is UniFi Network Application 7.4.156.

**Comment by frenck** [comment]
Yes, see also the release here: <https://github.com/hassio-addons/addon-unifi/releases>

The badge is in error, I will adjust that on the next iteration.

../Frenck
